shell/common/rasterizer.cc

```diff
@@ -181,7 +182,7 @@ sk_sp<SkImage> Rasterizer::DoMakeRasterSnapshot(
     // happen in case of software rendering.
     surface = SkSurface::MakeRaster(image_info);
   } else {
-    if (!surface_->MakeRenderContextCurrent()) {
```

Comment:
This particular change isn't as clear, `surface_->MakeRenderContextCurrent` is a much more clear verbiage than `context_switch->GetResult`

Reply:
We need to grab onto a switch object so it can live through out the scope. 
The switch object clears the GLContext on destruction.